### PR TITLE
Remove and redirect old installation guides

### PR DESCRIPTION
**Pull request by natke**
Remove the old individual install guides.
Update links to these old install guides
Add redirects

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/146802d0ede956a5b18b37ec3aeb4d1db9c4dc4e/articles/install-guide/update.md) | 86 | 65 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/0458192b-6d27-4772-bccd-9f662f8622a6_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/146802d0ede956a5b18b37ec3aeb4d1db9c4dc4e/articles/relnotes/index.md) | 75 | 50 | 156 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/67ceacf5-83b5-4873-b877-2d79ee597d43_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

File: articles/install-guide/update.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ The update steps depend on your development environment. Choose your environment
 
     ```bash
     iqsharp: 0.9.1909.3002
-    Jupyter Core: 1.1.14623.0
+    Jupyter Core: ...
```
Comment (cgranade) [inline articles/install-guide/update.md]:
```suggestion
    Jupyter Core: 1.1.18837.0
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/665d74b4fea982ee9a580f960ce6ac5e3f15d27b/articles/install-guide/update.md) | 86 | 65 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/817c6f7e-8eee-454f-bd7b-c994e87471d2_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/665d74b4fea982ee9a580f960ce6ac5e3f15d27b/articles/relnotes/index.md) | 75 | 50 | 156 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5ff74211-c791-4c09-892e-e2de76bc3fe8_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:

| Article | Acrolinx<br>Score | Lowest<br>Category<br>Score | Issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/install-guide/update.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/24d27f2bc5cc31e97753e4e1d889588d192439bd/articles/install-guide/update.md) | 86 | 65 | 8 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5d32adcd-cea7-495f-9335-2c7fbea2a3b9_report.html) | [:white_check_mark:](# "Successfully processed") |
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/24d27f2bc5cc31e97753e4e1d889588d192439bd/articles/relnotes/index.md) | 75 | 50 | 156 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/17fdee79-ef05-4436-8135-0a8b4af31d4e_report.html) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/en-us/help/contribute/contribute-acrolinx-vscode?branch=master)
- [False positives or issues](https://aka.ms/acrolinxbug)

**Pull request merged**